frontend/src/pages/modelRegistrySettings/ModelRegistrySettings.tsx

```diff
@@ -8,30 +8,48 @@ import {
   EmptyStateVariant,
 } from '@patternfly/react-core';
 import { PlusCircleIcon } from '@patternfly/react-icons';
+import { AreaContext } from '~/concepts/areas/AreaContext';
 import ApplicationsPage from '~/pages/ApplicationsPage';
-import useModelRegistriesBackend from '~/concepts/modelRegistrySettings/useModelRegistriesBackend';
+import RedirectErrorState from '~/pages/external/RedirectErrorState';
 import TitleWithIcon from '~/concepts/design/TitleWithIcon';
 import { ProjectObjectType } from '~/concepts/design/utils';
-import { ModelRegistrySelectorContext } from '~/concepts/modelRegistry/context/ModelRegistrySelectorContext';
+import useModelRegistriesBackend from '~/concepts/modelRegistrySettings/useModelRegistriesBackend';
 import { useContextResourceData } from '~/utilities/useContextResourceData';
 import { RoleBindingKind } from '~/k8sTypes';
 import ModelRegistriesTable from './ModelRegistriesTable';
 import CreateModal from './CreateModal';
 import useModelRegistryRoleBindings from './useModelRegistryRoleBindings';
 
 const ModelRegistrySettings: React.FC = () => {
+  const { dscStatus } = React.useContext(AreaContext);
+  const modelRegistryNamespace = dscStatus?.components?.modelregistry?.registriesNamespace;
   const [createModalOpen, setCreateModalOpen] = React.useState(false);
+
   const [modelRegistries, mrloaded, loadError, refreshModelRegistries] =
     useModelRegistriesBackend();
   const roleBindings = useContextResourceData<RoleBindingKind>(useModelRegistryRoleBindings());
-  const { refreshRulesReview } = React.useContext(ModelRegistrySelectorContext);
   const loaded = mrloaded && roleBindings.loaded;
 
   const refreshAll = React.useCallback(
-    () => Promise.all([refreshModelRegistries(), refreshRulesReview()]),
-    [refreshModelRegistries, refreshRulesReview],
```

Comment:
We can't remove this `refreshRulesReview()` call. That's what updates the registry selector after you create/delete a registry. Good call adding the `roleBindings.refresh` here though, although I'm curious what made you think to do that?